department-of-veterans-affairs/vets-website
# Pension: update service periods page

## Summary

- Update Branch of Service from short-text field to multi-select answer options, to match the PDF form. Answer options will be:
    - Army
    - Navy
    - Air Force
    - Coast Guard
    - Marine Corps
    - Space Force
    - USPHS
    - NOAA
- Match the start date and end date labels from the web form to match the PDF form. Labels should be:
    - Date initially entered active duty
    - Final release date from activity duty
- Remove the "Add another service period" button
- Remove the ability for user to enter multiple service periods

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the pension_form_enabled flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run vets-website and vets-api

## How to verify
1. Go to http://localhost:3001/pension/application/527EZ in your browser
2. Select 'Start your application without signing in'
3. Proceed to Chapter 2, military history
5. On the 'Service period' page, confirm that the page and questions don't allow you to add multiple periods; the Service Branch field allows the selection of multiple checkboxes; and the date field labels have been updated. 

## Related issue(s)

- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71705)
- [Link to Epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done


## Screenshots

| Before | After |
| ------- | ----------------|
| <img width="576" alt="Screenshot 2023-12-14 at 4 15 03 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/c243a0ba-18d8-4059-a3e0-75ff69a32f69"> |  <img width="548" alt="Screenshot 2023-12-14 at 1 53 12 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/69620480-2a29-4030-87e3-46fef536762b"> |

## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria

- [Link to GitHub Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71705)

### Quality Assurance & Testing


### Error Handling


### Authentication
